app/code/Magento/Cron/Model/Config/Backend/Product/Alert.php

```diff
@@ -82,22 +82,14 @@ public function afterSave()
         $cronExprString = join(' ', $cronExprArray);
```

Comment:
Not specifically a problem as join and implode are the same thing but if during this pr you have changed the code I would suggest we keep them the same between the two versions so I would request either revert that change in the other pr or make the same change here. The choice is yours. https://github.com/magento/magento2/pull/11567/files#diff-482e1a34ed1fa0e0e666785335496ef8R82

Reply:
@dmanners I didn't modify this function it was the phpcsfixer utility. I need to change it?

Reply:
@mrodespin ah ok, would be good to keep these 2 files a close as possible between 2.2 and 2.1 If you dont mind making that change that would be cool.